src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -0,0 +1,764 @@
+        defaultMap.put("UniProt_Region",                              UNKNOWN_VALUE_STRING);
+        defaultMap.put("UniProt_Site",                                UNKNOWN_VALUE_STRING);
+        defaultMap.put("UniProt_Natural_Variations",                  UNKNOWN_VALUE_STRING);
+        defaultMap.put("UniProt_Experimental_Info",                   UNKNOWN_VALUE_STRING);
+        defaultMap.put("GO_Biological_Process",                       UNKNOWN_VALUE_STRING);
+        defaultMap.put("GO_Cellular_Component",                       UNKNOWN_VALUE_STRING);
+        defaultMap.put("GO_Molecular_Function",                       UNKNOWN_VALUE_STRING);
+        defaultMap.put("COSMIC_overlapping_mutations",                UNKNOWN_VALUE_STRING);
+        defaultMap.put("COSMIC_fusion_genes",                         UNKNOWN_VALUE_STRING);
+        defaultMap.put("COSMIC_tissue_types_affected",                UNKNOWN_VALUE_STRING);
+        defaultMap.put("COSMIC_total_alterations_in_gene",            UNKNOWN_VALUE_STRING);
+        defaultMap.put("Tumorscape_Amplification_Peaks",              UNKNOWN_VALUE_STRING);
+        defaultMap.put("Tumorscape_Deletion_Peaks",                   UNKNOWN_VALUE_STRING);
+        defaultMap.put("TCGAscape_Amplification_Peaks",               UNKNOWN_VALUE_STRING);
+        defaultMap.put("TCGAscape_Deletion_Peaks",                    UNKNOWN_VALUE_STRING);
+        defaultMap.put("DrugBank",                                    UNKNOWN_VALUE_STRING);
+        defaultMap.put("ref_context",                                 UNKNOWN_VALUE_STRING);
+        defaultMap.put("gc_content",                                  UNKNOWN_VALUE_STRING);
+        defaultMap.put("CCLE_ONCOMAP_overlapping_mutations",          UNKNOWN_VALUE_STRING);
+        defaultMap.put("CCLE_ONCOMAP_total_mutations_in_gene",        UNKNOWN_VALUE_STRING);
+        defaultMap.put("CGC_Mutation_Type",                           UNKNOWN_VALUE_STRING);
+        defaultMap.put("CGC_Translocation_Partner",                   UNKNOWN_VALUE_STRING);
+        defaultMap.put("CGC_Tumor_Types_Somatic",                     UNKNOWN_VALUE_STRING);
+        defaultMap.put("CGC_Tumor_Types_Germline",                    UNKNOWN_VALUE_STRING);
+        defaultMap.put("CGC_Other_Diseases",                          UNKNOWN_VALUE_STRING);
+        defaultMap.put("DNARepairGenes_Activity_linked_to_OMIM",      UNKNOWN_VALUE_STRING);
+        defaultMap.put("FamilialCancerDatabase_Syndromes",            UNKNOWN_VALUE_STRING);
+        defaultMap.put("MUTSIG_Published_Results",                    UNKNOWN_VALUE_STRING);
+        defaultMap.put("OREGANNO_ID",                                 UNKNOWN_VALUE_STRING);
+        defaultMap.put("OREGANNO_Values",                             UNKNOWN_VALUE_STRING);
+    }
+
+    /**
+     * Initializes the output map
+     */
+    private void initializeOutputFieldNameMap() {
```

Comment:
You may want to drive this from a configuration file.  Even if not, you should refactor out some constants (at least for the actual MAF Field names)

Reply:
Agreed - again, I'd like to have a config file, but I wanted to have some hard-coded defaults for when there is no config file given.